### DISCUSSION TITLE
Change e-mail

### DISCUSSION THREAD

**Discussion by Isa358** [aclap-dev/video-downloadhelper#246] [License & Purchase requests]
### What's your issue?

I just noticed that there was an error in the email when purchasing the license. How to correct it? The e-mail correct is isamyr20@gmail.com.

### The 10 first digits (only 10) of your license number or email address

isasmyr20@gmail.com

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by nala90** [comment]
Coloquei o e-mail errada, digitei alan0553@gmail.com o email correto é alan0553@outlook.com.
Não fiz a instalação da licença, preciso de ajuda.

**Comment by jcfrog** [comment]
Hi,
@Isa358 and @nala90 I sent you an email.

**Reply by nala90** [reply]
OK.
